src/main/java/httpserver/http/route/RouteHandler.java

```diff
@@ -0,0 +1,43 @@
+package httpserver.http.route;
+
+import httpserver.http.*;
+import httpserver.http.request.RequestSplitter;
+import httpserver.http.response.Response;
+
+public class RouteHandler {
+    private final String request;
+
+    public RouteHandler(String request) {
+        this.request = request;
+    }
+
+    public String getResponse() {
+        var requestMethod = new RequestSplitter(request).getRequestMethod();
+        var requestPath = new RequestSplitter(request).getRequestPath();
+
+        if (requestMethod.equals(RequestMethod.GET.toString())
+                && requestPath.equals("/simple_get")) {
+            return simpleGetResponse();
+        } else if (requestMethod.equals(RequestMethod.OPTIONS.toString())
+                && requestPath.equals("/method_options")) {
+            return methodOptionsResponse();
+        }
+        return notFound();
+    }
+
+    private String simpleGetResponse() {
+        return new Response(ProtocolVersion.HTTP_1_1, StatusCode.OK)
```

Comment:
Might be a nice use case for a Builder, which could look like:

```
new ResponseBuilder()
  .protocolVersion(ProtocolVersion.HTTP_1_1)
  .statusCode(StatusCode.OK)
  .build();
```

Reply:
That's great. I'll look into this, thank you.